src/Serilog.Sinks.Email/Sinks/Email/MailKitSecureSocketOptions.cs

```diff
@@ -0,0 +1,40 @@
+﻿#if MAIL_KIT
+
+namespace Serilog.Sinks.Email.Sinks.Email
+{
```

Comment:
I believe the goal of adding this enum was to hide dependency on MailKit. So maybe it should be named just "SecureSocketOptions"?